packages/@react-spectrum/textfield/stories/Textfield.stories.js

```diff
@@ -140,6 +140,21 @@ storiesOf('TextField', module)
   )
   .add('custom width small, labelPosition: side',
     () => render({icon: <Info />, validationState: 'invalid', width: '30px', labelPosition: 'side'})
+  )
+  .add('test: isReadOnly = true, value = this is read only',
+    () => render({isReadOnly: true, value: 'this is read only'})
+  )
+  .add('test: isReadOnly, value = "" ',
+    () => render({isReadOnly: true, value: ''})
+  )
+  .add('test: isReadOnly, long text',
+  () => render({isReadOnly: true, value: 'foo  '.repeat(10)})
+  )
```

Comment:
```suggestion
  .add('test: isReadOnly, long text',
  () => render({isReadOnly: true, value: 'foo  '.repeat(50)})
  )
```
Just so we can test a more extreme case